src/Shared/E2ETesting/BrowserFixture.cs

```diff
@@ -2,54 +2,28 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+using System.Collections.Generic;
 using System.Linq;
 using System.Reflection;
+using System.Threading.Tasks;
 using OpenQA.Selenium;
 using OpenQA.Selenium.Chrome;
 using OpenQA.Selenium.Remote;
 using Xunit.Abstractions;
-using Xunit.Sdk;
 
 namespace Microsoft.AspNetCore.E2ETesting
 {
-    public class BrowserFixture : IDisposable
```

Comment:
Why not leave this as disposable? Since the Fixture is holding a reference to the Browser and handing it out to anyone who asks for one it seems to me the Fixture needs to be responsible for disposing of the browser. If a browser that's been handed out to two places gets disposed in one test the second would (I assume) crash.

Reply:
That's not the case. Each test gets its own browser.

Reply:
If that's the case why do we have `GetOrCreateBrowserAsync`? Or do you mean that each test gets its own BrowserFixture?

Reply:
This is back to where it was.